pkg/cmd/kube-router.go

```diff
@@ -183,8 +187,39 @@ func (kr *KubeRouter) Run() error {
 	}
 
 	if kr.Config.RunFirewall {
+		iptablesCmdHandlers := make(map[v1core.IPFamily]utils.IPTablesHandler, 2)
+		ipSetHandlers := make(map[v1core.IPFamily]utils.IPSetHandler, 2)
+
+		if kr.Config.EnableIPv4 {
+			iptHandler, err := iptables.NewWithProtocol(iptables.ProtocolIPv4)
+			if err != nil {
+				return fmt.Errorf("failed to create iptables handler: %w", err)
```

Comment:
You could use `errors.New("failed to create iptables handler: " + err.Error())` and avoid importing `fmt`, or? I was just checking how errors are created in this gofile

Reply:
`errors.New` is not wrapping the original error, while `fmt.Errorf("%w", err)` keeps the reference to the original error.

https://golangbyexample.com/wrapping-and-unwrapping-error-golang/